### ISSUE TITLE
Debug log formatting

### ISSUE THREAD

**Issue by a1lu** [schollz/croc#353]
**Please try to [download the latest version](https://github.com/schollz/croc/releases/latest) of croc before reporting a bug!**

**Describe the bug**
Some debug messages are not formatted correctly if run with `croc --debug`
```
[debug]	16:44:44 croc.go:675: exchanged header message
securing channel...[debug]	16:44:44 croc.go:687: ready
[debug]	16:44:44 message.go:46: writing pake message
```

It looks like this comes from the different handling of debug messages. 

```
log.Debug("ready")
```
vs

```
fmt.Fprintf(os.Stderr, "\rsecuring channel...")
```

vs

``` 
log.Debugf("writing %s message", m.Type)
```

**To Reproduce**

```
croc send --text "test"
croc --debug <phrase>
```

**Expected behavior**
[debug] should be at the start of the line

**Version**
croc version v8.6.12-c373b38

**Additional context**

**Comment by schollz** [comment]
Happy to accept a PR for this.

**Comment by a1lu** [comment]
I got reminded about this issue through the github-actions.
I checked the code and guess its by design, that
```
	fmt.Fprintf(os.Stderr, "connecting...")
```
and
```
	fmt.Fprintf(os.Stderr, "\rsecuring channel...")
```
don't print a newline, because the latter overwrites the first one using `\r`.
So the formatting is fine until `--debug` is used. But I would not introduce a `\n` if `--debug` is set just to have a bit cleaner output.
So I would /close it too.
